linux/utils/test_setup.yml

```diff
@@ -108,12 +108,11 @@
 - include_tasks: check_guest_os_gui.yml
   when: guest_os_with_gui is undefined
 
-- block:
-    - include_tasks: get_guest_ovt_version_build.yml
-
-    - name: "Print open-vm-tools version and build"
-      debug: var=vmtools_info_from_vmtoolsd
-  when: vmtools_check and vmtools_is_installed|bool
+- include_tasks: get_guest_ovt_version_build.yml
+  when:
+    - vmtools_check
```

Comment:
I think here we can just check if "vmtools_is_installed", since above `vmtools_check` is in  'or' condition that means 'vmtools_check' is not the decisive condition, here it is.